owtf.py

```diff
@@ -255,4 +255,7 @@ def main(args):
 
 
 if __name__ == "__main__":
+    if os.getcwd().find(os.path.dirname(os.path.abspath(sys.argv[0])))==0:
+        usage("Start owtf in a different directory. 'cd path/to/pentest/directory; ./path/to/owtf.py'")
+        exit(-1)
```

Comment:
Use `sys.exit` here. (exit is a helper for the interactive shell - sys.exit is intended for use in programs.)